src/scheduler/base.py

```diff
@@ -72,12 +78,17 @@ def jobstore(self):
 
     @classmethod
     def run_job(cls, job_class_path, job_id, db_url, *args, **kwargs):
+        datasource_id = args[0]
+
         jobstore = MsiJobStore(url=db_url)
+        last_run = jobstore.get(datasource_id).get('created_at')
+        print(last_run)
```

Comment:
remove

Reply:
still important as I'm not 100% sure this works.

Reply:
Removed